portfolio/components/ui/Figures/SectionEmbeddingExplorer/SectionEmbeddingExplorer.module.css

```
@@ -0,0 +1,664 @@
+
+.receiptFooter {
+  height: 20px;
+  border-top: 1px solid rgba(var(--text-color-rgb), 0.09);
+  border-bottom: 0;
+  font-size: 0.5rem;
+  text-transform: uppercase;
+}
+
+.currentReceipt {
+  --row-height: 21px;
+  top: 3.5rem;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)